packages/docker-node/src/interface.ts

```diff
@@ -0,0 +1,15 @@
+import { config } from '@ckb-lumos/lumos'
+
+export interface CKBNode {
+  url: string
+  port: string
+  host: string
+  lumosConfig: config.Config
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  start(params: any): void
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  stop(params: any): void
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  deployScripts(params: any): void
```

Comment:
Could we provide some necessary parameters for these functions?

Reply:
> Could we provide some necessary parameters for these functions?

Each implementation of the `Node` may has different parameters to start or stop, so I use the `any` as the parameter type here.

Reply:
Can we resolve this conversation?

Reply:
> Can we resolve this conversation?

I have no idea this time, may I can have a consideration and make it possible later in another PR.